bin/generate.sh

```diff
@@ -0,0 +1,7 @@
+set -e
+
+if [ -z "$1" ] || [ -z "$2" ]; then
```

Comment:
The representer [takes three arguments](https://github.com/exercism/automated-analysis/blob/master/docs/representers/interface.md):

- The slug of the exercise (e.g. two-fer).
- A path to a directory containing the submitted file(s) (with a trailing slash).
- A path to an output directory (with a trailing slash). This directory is writable.

Reply:
Probably not. I assume `joinPath` will do that for you when combining paths.

Reply:
Not sure.
I just use the `/` operator